### ISSUE TITLE
Material Carousel a11y improvements

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4237]
From #4194 Sample 5.2 on a Material theme homepage:

> Carousel not identified as such. Navigating by heading gets to visible item on carousel, as an H2. But as this is a carousel, this keeps changing making it awkward when navigating past it, then back to find previous heading has changed.

**Comment by joemull** [comment]
We think we should move all themes to an accessible, ES6-based carousel tool like glide.js, which we used for Hourglass.

**Comment by StephDriver** [comment]
When updating this, consider WCAG 2.1 Keyboard Accessible and WCAG 2.2 Enough Time.  Neither are currently supported by the carousel.

**Comment by StephDriver** [comment]
adding that this can also be a particular annoyance when there is only one image - as the carousel still moves, and the page background shows for a split second.  If the image is has a lot of black background, and the page is white, this makes it 'flash' between the two extremes.  The carousel doesn't move fast enough for this to contravene the 3 flash in a second threshold, but it does move swiftly enough for this to be visually uncomfortable.

**Comment by StephDriver** [comment]
Further testing with screen-reader on #4977 showed the material navigation was not screen reader compatible, despite keyboard compatibility work and aria.  So decision was that screen reader users will be signposted to accessibility mode instead, and there will be no further work specifically aimed at material screen reader compatibility.  Good practice in terms of landmarks, headings, alt text etc, will still continue, just nothing that requires specific debugging of screen reader issues.
